test/testcommon/src/main/java/io/pravega/test/common/NoOpScheduledExecutor.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.common;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * A fake implementation of ScheduledExecutorService that does nothing. An INSTANCE of this class may be used as a
+ * placeholder for unused scheduled executor in tests.
+ */
+public class NoOpScheduledExecutor implements ScheduledExecutorService {
+
+    private static final NoOpScheduledExecutor INSTANCE = new NoOpScheduledExecutor();
+
+    public static NoOpScheduledExecutor get() {
+        return INSTANCE;
+    }
+
+    @Override
+    public ScheduledFuture<?> schedule(Runnable runnable, long l, TimeUnit timeUnit) {
+        return null;
+    }
+
+    @Override
+    public <V> ScheduledFuture<V> schedule(Callable<V> callable, long l, TimeUnit timeUnit) {
+        return null;
+    }
+
+    @Override
+    public ScheduledFuture<?> scheduleAtFixedRate(Runnable runnable, long l, long l1, TimeUnit timeUnit) {
+        return null;
+    }
+
+    @Override
+    public ScheduledFuture<?> scheduleWithFixedDelay(Runnable runnable, long l, long l1, TimeUnit timeUnit) {
+        return null;
+    }
+
+    @Override
+    public void shutdown() {
+
+    }
+
+    @Override
+    public List<Runnable> shutdownNow() {
+        return null;
+    }
+
+    @Override
+    public boolean isShutdown() {
+        return false;
```

Comment:
Should return true if you invoked `shutdown`

Reply:
Fixed.